examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -20,34 +21,48 @@ if (location.hash.length === 0) {
 const containerId = location.hash.substring(1);
 document.title = containerId;
 
-const dataObjectId = "dice";
+// Define the configuration of our Container.
+// This includes the DataObjects we support and any initial DataObjects we want created
+// when the container is first created.
+export const containerConfig = {
+    name: "dice-roller-container",
+    initialObjects: {
+        /* [id]: DataObject */
+        kvp: KeyValueDataObject,
+        map: SharedMap,
+    },
+};
 
 async function start(): Promise<void> {
-    // Define the configuration of our Container.
-    // This includes the DataObjects we support and any initial DataObjects we want created
-    // when the container is first created.
-    const containerConfig = {
-        dataObjects: [KeyValueDataObject],
-        initialDataObjects: {
-            /* [id]: DataObject */
-            [dataObjectId]: KeyValueDataObject,
-        },
-    };
     // Get or create the document depending if we are running through the create new flow
     const fluidContainer = createNew
         ? await Fluid.createContainer(containerId, containerConfig)
         : await Fluid.getContainer(containerId, containerConfig);
 
     // We now get the DataObject from the container
-    const keyValueDataObject = await fluidContainer.getDataObject<KeyValueDataObject>(dataObjectId);
+    const keyValueDataObject = fluidContainer.initialObjects.kvp as KeyValueDataObject;
 
     // Our controller manipulates the data object (model).
     const diceRollerController = new DiceRollerController(keyValueDataObject);
     await diceRollerController.initialize(createNew);
 
     // We render a view which uses the controller.
-    const div = document.getElementById("content") as HTMLDivElement;
-    renderDiceRoller(diceRollerController, div);
+    const contentDiv = document.getElementById("content") as HTMLDivElement;
+    const div1 = document.createElement("div");
+    contentDiv.appendChild(div1);
+    renderDiceRoller(diceRollerController, div1);
+
+    // We now get the DataObject from the container
+    const sharedMap = fluidContainer.initialObjects.map as SharedMap;
```

Comment:
Is the purpose here to show the flexibility of data objects here?

Reply:
It's to show/test DDS usage. I copied and pasted the comment unintentionally